recipe/meta.yaml

```diff
@@ -36,6 +36,7 @@ build:
 
 requirements:
   build:
+    # gobject-introspection 1.* requires setuptools at runtime, but
     # setuptools >= 65.0.0 is lacking the msvccompiler module in distutils.
     # https://github.com/pypa/setuptools/pull/3505
     - setuptools <65
```

Comment:
Can we patch this from `msvccompiler` to `_msvccompiler`? It looks like in modern setuptools it exists but it obfuscated. I'm not sure if it's worth the effort though? https://github.com/pypa/setuptools/blob/main/setuptools/_distutils/_msvccompiler.py

Reply:
Oh It's a build time dep - nevermind! Need more caffiene!